Button/style.css

```diff
@@ -2,6 +2,7 @@
 
 .button {
   display: inline-block;
+  margin: 0;
```

Comment:
Interesting thought, @hharnisc! I think this change in `buffer-web` might be quite an interesting one with the legacy `buttons.less` and `Button.less` files. I wonder... with the goal of this change being to ensure our `PostFooter`s are the same height regardless of if they have a `Button` in or not, perhaps we could apply some fixed height to `Button`s contained within `PostFooter`? 🤔 I definitely think this is a good call to remove this forced `margin` here either way! I totally didn't think of the global impact here. 😅